scipy/stats/tests/test_distributions.py

```diff
@@ -926,7 +929,7 @@ def test_rvs_negative_p(self):
 
     def test_invgauss(self):
         # test that invgauss is special case
-        ig = stats.geninvgauss.rvs(size=1500, p=-0.5, b=1, random_state=1234)
+        ig = stats.geninvgauss.rvs(size=1500, p=-0.5, b=1, random_state=1464878613)
```

Comment:
Why does this use an integer and not a `Generator` instance?

Reply:
Because it did before. I'm not abolishing use of RandomState in this PR; just removing `seed`.